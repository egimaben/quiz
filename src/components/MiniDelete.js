import React from 'react';
class MiniDelete extends React.Component {
  handleClick = (e) => {
    e.preventDefault();
    const { callback, item } = this.props;
    callback(item);
  }
  render() {
    return (<React.Fragment><button onClick={this.handleClick}>x</button><div>{this.props.item}</div></React.Fragment>);
  }
}
export default MiniDelete;


