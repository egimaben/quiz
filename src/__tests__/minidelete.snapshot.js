'use strict'
import React from 'react';
import renderer from 'react-test-renderer';
import MiniDelete from '../components/MiniDelete';
it('displays delete button component correctly',()=>{
    //when should not show continue button
    const tree = renderer.create(
        <MiniDelete item='item 1'/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
})