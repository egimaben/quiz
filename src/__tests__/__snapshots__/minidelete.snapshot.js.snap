// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays delete button component correctly 1`] = `
Array [
  <button
    onClick={[Function]}
  >
    x
  </button>,
  <div>
    item 1
  </div>,
]
`;
